<ui version="4.0" >
 <class>dlg_adl_prop</class>
 <widget class="QDialog" name="dlg_adl_prop" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Properties</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Search</string>
       </property>
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>4</number>
        </property>
        <property name="topMargin" >
         <number>4</number>
        </property>
        <property name="rightMargin" >
         <number>4</number>
        </property>
        <property name="bottomMargin" >
         <number>8</number>
        </property>
        <item row="0" column="0" >
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>String:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edt_string" />
          </item>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmb_searchtype" />
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>FileSize</string>
       </property>
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>4</number>
        </property>
        <property name="topMargin" >
         <number>4</number>
        </property>
        <property name="rightMargin" >
         <number>4</number>
        </property>
        <property name="bottomMargin" >
         <number>8</number>
        </property>
        <item row="0" column="0" >
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Min:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edt_minsize" >
            <property name="minimumSize" >
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="inputMask" >
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Max:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edt_maxsize" >
            <property name="minimumSize" >
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmb_sizetype" />
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Destination Directory:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_destdir" />
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QCheckBox" name="cb_enabled" >
           <property name="text" >
            <string>Enabled</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_dlmatches" >
           <property name="text" >
            <string>Download matches</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
